dockerfiles/cosmos-exporter/v0.3.0/Dockerfile

```
@@ -0,0 +1,8 @@
+FROM alpine:3.16
+EXPOSE 9300
+
+RUN wget -q https://github.com/solarlabsteam/cosmos-exporter/releases/download/v0.3.0/cosmos-exporter_0.3.0_Linux_x86_64.tar.gz &&\
+    tar xvfz cosmos-exporter* -C /usr/bin
```

```suggestion
RUN wget -q https://github.com/solarlabsteam/cosmos-exporter/releases/download/v0.3.0/cosmos-exporter_0.3.0_Linux_x86_64.tar.gz -O - |\
    tar -zxvf - -C /usr/bin
```

We can one-line it by streaming the `wget` output to `tar`, the purpose here is to avoid having the `cosmos-exporter_0.3.0_Linux_x86_64.tar.gz` file on the filesystem. Another way could be to remove it after extraction.